src/neo/SmartContract/InteropService.Contract.cs

```diff
@@ -148,13 +144,25 @@ private static bool Contract_CallEx(ApplicationEngine engine, UInt160 contractHa
                 UInt160 callingScriptHash = state.ScriptHash;
                 CallFlags callingFlags = state.CallFlags;
 
-                ExecutionContext context_new = engine.LoadScript(contract.Script, 1);
+                ContractMethodDescriptor md = contract.Manifest.Abi.GetMethod(method);
+                if (md is null) return false;
+                int rvcount = md.ReturnType == ContractParameterType.Void ? 0 : 1;
+                ExecutionContext context_new = engine.LoadScript(contract.Script, rvcount);
                 state = context_new.GetState<ExecutionContextState>();
                 state.CallingScriptHash = callingScriptHash;
                 state.CallFlags = flags & callingFlags;
 
-                context_new.EvaluationStack.Push(args);
-                context_new.EvaluationStack.Push(method);
+                if (NativeContract.IsNative(contractHash))
```

Comment:
Native is native?
Is initialized?

Reply:
Native contracts have no code, so you cannot find methods by offset. Therefore, `Contract.Call` needs to check whether the called contract is native.